presto-docs/src/main/sphinx/release/release-0.258.rst

```diff
@@ -0,0 +1,44 @@
+=============
+Release 0.258
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix issues in SQL functions where query might fail with compiler error when function implementation includes lambda variable that has the same name as column names or function input names.
+* Fixes a bug in Parquet dereference pushdown. The issue causes column name not picked up properly, resulting in inconsistent query result in certain cases.
+* Add Cauchy distribution CDF :func:`cauchy_cdf` and inverse CDF :func:`inverse_cauchy_cdf` functions to MathFunctions.
+* Add SQL functions :func:`array_dupes` and :func:`array_as_dupes`.
+* Add additional details to memory exceeded error messages to simplify debugging. Disabled by default. Can be enabled by setting the ``verbose_exceeded_memory_limit_errors_enabled`` session property to ``true``.
+* Allow dynamic filtering for large number of collected values by converting the predicate into a range.
+* Support dynamic filtering with comparison operators.
```

Comment:
what is dynamic filtering?

Reply:
@viczhang861 @rongrong can one of you clarify this given you have approved the pr?

Reply:
This looks like a cherry pick from trino: https://github.com/prestodb/presto/pull/16401
And from the trino PR some details:

_Following #52, we would like to support dynamic filtering for cross joins (e.g. having inequality predicates in the join criteria). For example, the following query should benefit from this feature:_

```
SELECT
	count() 
FROM
	orders o, part p 
WHERE
	p.comment = "Very expensive part" AND 
	o.totalprice > p.retailprice 
```